raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -679,7 +680,7 @@ def test_refund_transfer_no_more_routes():
     )
     assert iteration.new_state, 'payment task should not be deleted at this lock expired'
     # should not be accepted
-    assert not events.must_contain_entry(iteration.events, SendProcessed, {})
+    assert search_for_item(iteration.events, SendProcessed, {}) is None
```

Comment:
I did change some of `not ...` to `... is None`, but to limit the changes and make reviews easier I stopped